README.md

```diff
@@ -1,2 +1,99 @@
-# tw-academy-s9
-Activity for session 9
+# **Sample Style Guide**            
+### Technical Writing Academy - May 2021
+
+### **Table of contents**
+
+#### [Bold](Bold)
+#### [Italics](Italics)
+#### [Fixed Font](Fixed-Font)
+#### [Abbreviations](Abbreviations)
+#### [Capitalization](Capitalization)
+#### [Acronyms](Acronyms)
+#### [Commas in Lists](Commas-in-Lists)
+#### [Numbers](Numbers)
+#### [Contractions](Contractions)
+#### [Apostrophes](Apostrophes)
+
+### **Bold**
+
+Use bold to highlight button names, fields names, menus, radio buttons, or checkboxes. For
+example:
+* Click the **Create** button to start the process
+* Enter your IP address in the **IP Address** field.
+
+### **Italics**
+
+Use italics to emphasize a word or group of words. For example:
+* _At least_ two reviews are required before you check in your code
+* You must _not_ enter without your ID card.
+> **IMPORTANT:** Use this style sparingly to avoid diluting its strength
+
+### **Fixed Font**
+
+Use a fixed font to highlight:
+* **Commands or pieces of code.** For example:  
+Run the `npm install text-cli` command to install the utility.
+* **File or folder names.** For example:  
+Read the <span style="color:#fff; font-family: 'CourierNew'; font-size: 1em;">README.md</span> file before you proceed.
+
+### **Abbreviations**
+
+Never use the following abbreviations in a formal document:
+
+|               |               |
```

Comment:
I see that the original document did not include headers, but you can add them. Otherwise, you only see empty cells.